apigen.neon

```diff
@@ -1,6 +1,6 @@
 source: ./
 
-destination: docs.woothemes.com/images/sensei-apidocs
+destination: docs.woocommerce.com/images/sensei-apidocs
```

Comment:
Thanks @jeherve the move has been finalized (docs.woocommerce.com no live). I've also sent in another PR for the docs static files: https://github.com/Automattic/docs.woothemes.com/pull/113 . These are independent as docs are not auto generated.